vendor/engines/bulk_email/app/views/bulk_email/bulk_email/_fields.html.haml

```diff
@@ -26,3 +26,6 @@
   .row
     .span8
       = f.submit text("shared.search"), class: "btn"
+      - if params[:return_path].present?
+        = link_to t("shared.cancel"), params[:return_path]
```

Comment:
I'm updating this now, but I'm curious if you're still seeing that "Search" button way off on the right. I'm seeing it just to the left of the "Cancel" link right now (Chrome, Safari, and FF).